app/controllers/s3_multipart/uploads_controller.rb

```diff
@@ -28,50 +28,53 @@ def update
 
     private
 
-      def sign_batch
-        begin
-          response = Upload.sign_batch(params)
-        rescue => e
-          logger.error "EXC: #{e.message}"
-          response = {
-            error_message: t("s3_multipart.errors.update"),
-            error: e.message
-          }
-        ensure
-          render :json => response
-        end
+    def sign_batch
```

Comment:
I think the private methods should remain indented to match the style of the rest of this repo

Reply:
👎 but ok! haha